debian/swtpm-libs.install

```diff
@@ -1,2 +1 @@
-#! /usr/bin/dh-exec
```

Comment:
Since this file doesn't use any `dh-exec` extensions (e.g. renaming files as they're installed) and `swtpm` doesn't even depend on `dh-exec`, this shouldn't be here.